content/200-orm/500-reference/100-prisma-schema-reference.mdx

```diff
@@ -2881,11 +2881,11 @@ model Post {
 
 <AlgoliaTerm name="apiReference" value="@ignore" />
 
-Add `@ignore` to a field that you want to exclude from Prisma Client (for example, a field that you do not want Prisma users to update). Ignored fields are excluded from the generated Prisma Client. The model's `create` method is disabled when doing this for _required_ fields with no `@default` (because the database cannot create an entry without that data).
+Add `@ignore` to a field that you want to exclude from Prisma Client (for example, a field that you do not want Prisma ORM users to update). Ignored fields are excluded from the generated Prisma Client. The model's `create` method is disabled when doing this for _required_ fields with no `@default` (because the database cannot create an entry without that data).
```

Comment:
```suggestion
Add `@ignore` to a field that you want to exclude from Prisma Client (for example, a field that you do not want Prisma Client users to update). Ignored fields are excluded from the generated Prisma Client. The model's `create` method is disabled when doing this for _required_ fields with no `@default` (because the database cannot create an entry without that data).
```